modules/exploits/linux/http/control_web_panel_login_cmd_exec.rb

```diff
@@ -0,0 +1,128 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'rex/stopwatch'
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'CWP login.php Unauthenticated RCE',
+        'Description' => %q{
+          Control Web Panel versions < 0.9.8.1147 are vulnerable to
+          unauthenticated OS command injection. Successful exploitation results
+          in code execution as the root user. The results of the command are not
```

Comment:
The `not not` strikes again!

Reply:
Yeah, that's what I get for copy-pasting 😅  Thanks for catching this!